brainglobe_stitch/image_mosaic.py

```diff
@@ -340,3 +346,366 @@ def read_big_stitcher_transforms(self) -> None:
         for tile in self.tiles:
             stitched_position = stitched_translations[tile.id]
             tile.position = stitched_position
+
+    def fuse(
+        self,
+        output_file_name: str = "fused.zarr",
```

Comment:
I went back and forth on this, this was partially for my own laze when I was writing this function. I'll remove it and if I feel the default is needed I'll place it in the UI instead of here.